metaflow/plugins/kubernetes/kube_utils.py

```diff
@@ -23,3 +23,26 @@ def parse_cli_options(flow_name, run_id, user, my_runs, echo):
             raise CommandException("A previous run id was not found. Specify --run-id.")
 
     return flow_name, run_id, user
+
+
+def qos_requests_and_limits(qos_class: str, cpu: int, memory: int):
+    "return resource requests and limits for the kubernetes pod based on the given QoS Class"
+    # Determine the requests and limits to define chosen QoS class
+    qos_limits = {}
+    qos_requests = {}
+    if qos_class == "Guaranteed":
+        # Guaranteed - has both cpu/memory limits. requests not required, as these will be inferred.
+        qos_limits = {
+            "cpu": str(cpu),
+            "memory": "%sM" % str(memory),
```

Comment:
what about storage?

Reply:
just ephemeral-storage. gpu's always need to specified with both requests and limits. also, can you verify airflow too?

Reply:
handling storage as well now. For the `BestEffort` case, I figure we keep the ephemeral-storage in the requests, or do we omit it completely?